<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="editor">
      <property name="autoFormatting">
       <set>QTextEdit::AutoBulletList</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="separator"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionSalvar_como"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionRecortar"/>
    <addaction name="actionCopiar"/>
    <addaction name="actionColar"/>
   </widget>
   <widget class="QMenu" name="menuFormatar">
    <property name="title">
     <string>Formatar</string>
    </property>
    <addaction name="actionNegrito"/>
    <addaction name="actionIt_lico"/>
    <addaction name="actionSublinhado"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuFormatar"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNovo"/>
   <addaction name="separator"/>
   <addaction name="actionAbrir"/>
   <addaction name="actionSalvar"/>
   <addaction name="separator"/>
   <addaction name="actionNegrito"/>
   <addaction name="actionIt_lico"/>
   <addaction name="actionSublinhado"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNegrito">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="botoes.qrc">
     <normaloff>:/botoes/negrito</normaloff>:/botoes/negrito</iconset>
   </property>
   <property name="text">
    <string>Negrito</string>
   </property>
  </action>
  <action name="actionIt_lico">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="botoes.qrc">
     <normaloff>:/botoes/italico</normaloff>:/botoes/italico</iconset>
   </property>
   <property name="text">
    <string>It√°lico</string>
   </property>
  </action>
  <action name="actionSublinhado">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="botoes.qrc">
     <normaloff>:/botoes/sublinhado</normaloff>:/botoes/sublinhado</iconset>
   </property>
   <property name="text">
    <string>Sublinhado</string>
   </property>
  </action>
  <action name="actionRecortar">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="botoes.qrc">
     <normaloff>:/botoes/recortar</normaloff>:/botoes/recortar</iconset>
   </property>
   <property name="text">
    <string>Recortar</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="botoes.qrc">
     <normaloff>:/botoes/copiar</normaloff>:/botoes/copiar</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionColar">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="botoes.qrc">
     <normaloff>:/botoes/colar</normaloff>:/botoes/colar</iconset>
   </property>
   <property name="text">
    <string>Colar</string>
   </property>
  </action>
  <action name="actionNovo">
   <property name="icon">
    <iconset resource="botoes.qrc">
     <normaloff>:/botoes/novo</normaloff>:/botoes/novo</iconset>
   </property>
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="botoes.qrc">
     <normaloff>:/botoes/abrir</normaloff>:/botoes/abrir</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="icon">
    <iconset resource="botoes.qrc">
     <normaloff>:/botoes/salvar</normaloff>:/botoes/salvar</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="botoes.qrc">
     <normaloff>:/botoes/sair</normaloff>:/botoes/sair</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionSalvar_como">
   <property name="text">
    <string>Salvar como...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="botoes.qrc"/>
 </resources>
 <connections/>
</ui>
